action.yml

```diff
@@ -21,9 +21,18 @@ inputs:
   sha256:
     description: 'SHA256 hash value to check against'
     required: false
+  authentication:
+    description: '`None`|`Basic` to indicate authentication type'
+    required: false
+  username:
+    description: 'Used in `Basic` authentication'
+    required: false
+  password:
+    description: 'Used in `Basic` authentication'
+    required: false
 outputs:
   file-path:
     description: 'The full path to the downloaded file'
 runs:
-  using: 'node16'
+  using: 'node12'
```

Comment:
Hi!  
Maybe this is an unnecessary modification?

Reply:
I'm fine changing it back - It is just that making it node12 makes it compatible with https://github.com/nektos/act out of the box.

Reply:
And there are no downsides AFAICT...

Reply:
> It is just that making it node12 makes it compatible with https://github.com/nektos/act out of the box.

Oh, I didn't know "nektos/act", thank you.  
And I've found that the node16 support has been added in nektos/act#922 , did you try that?

> And there are no downsides AFAICT...

Well, yes, but it's not relevant to this PR. IMHO, to keep git-blame clean, we need to make changes as small as possible.

Reply:
Makes total sense. Works for me! Updating now. Thanks!